SqlReplay.Console/Runner.cs

```diff
@@ -181,14 +195,14 @@ private async Task RunEventsAsync(Run run)
                 {
                     await Task.Delay(timeToDelay);
                 }
-                Console.WriteLine("Starting bucket: " + bucketTimestamp);
+                Console.WriteLine(DateTime.Now + " - Starting bucket: " + bucketTimestamp + $" - timeToDelay = {timeToDelay}");
```

Comment:
There are a few of these lines that would be a lot easier to read if they didn't mix string interpolation (`$`) with the `+` operator. I'd say stick with one or the other for the extent of the statement. _(my preference would be interpolation :) )_

Reply:
good call, I cleaned those up